src/runtime/manifest-ast-nodes.ts

```diff
@@ -163,13 +163,26 @@ export interface Particle extends BaseNode {
   slots?: ParticleSlot[];    // not used in RecipeParticle
   description?: Description;  // not used in RecipeParticle
   hasParticleArgument?: boolean;  // not used in RecipeParticle
+  trustChecks?: ParticleTrustCheck[];
+  trustClaims?: ParticleTrustClaim[];
 
   // fields in RecipeParticle only
   ref?: ParticleRef | '*';
   connections?: RecipeParticleConnection[];
   slotConnections?: RecipeParticleSlotConnection[];
 }
 
+export interface ParticleTrustClaim extends BaseNode {
+  kind: 'particle-trust-claim';
+  handle: string;
+  trustTag: string;
+}
+
+export interface ParticleTrustCheck extends BaseNode {
+  kind: 'particle-trust-check';
```

Comment:
I think there's an expectation that anything with a kind should have a location as well, so that error messages can correctly be generated.

Reply:
location is a property on BaseNode, so this is already done